src/common/extraction/extractNxGraph.ts

```diff
@@ -0,0 +1,70 @@
+import * as fs from "fs"
+import * as path from "path"
+import { Graph } from "./graph"
+
+type Nodes = {
+	[index: string]: [Outgoing]
+}
+
+type Outgoing = {
+	source: string
+	target: string
+}
+
+export function extractNxGraph(rootFolder?: string): Graph {
+	const workspaceRoot = rootFolder ?? path.dirname(guessLocationOfNxConfig()!)
```

Comment:
Would it be better to throw a human-readable error if `guessLocationOfNxConfig()` is undefined? At the moment, dirname will throw a TypeError

Reply:
Good catch! I fixed that.